docs/GlobalAPI.md

```diff
@@ -256,6 +256,81 @@ describe('binaryStringToNumber', () => {
 });
 ```
 
+### `describe.each(table)(name, fn)`
+
+Use `describe.each` if you keep duplicating the same test suites with different
+data. `describe.each` allows you to write the test suite once and pass data in.
+
+`describe.each` is available with two APIs:
+
+#### 1. `describe.each(table)(name, fn)`
+
+* `table`: `Array` of Arrays with the arguments that are passed into the `fn`
+  for each row.
+* `name`: `String` the title of the test suite, use `%s` to positionally inject
+  test data into the suite title.
+* `fn`: `Function` the suite of tests to be ran, this is the function that will
+  receive the parameters in each row as function arguments.
+
+Example:
+
+```js
+describe.each([[1, 1, 2], [1, 2, 3], [2, 1, 3]])(
+  '.add(%s, %s)',
+  (a, b, expected) => {
+    test(`returns ${expected}`, () => {
+      expect(a + b).toBe(expected);
+    });
+
+    test('does not mutate first arg', () => {
+      a + b;
```

Comment:
what does this line do?

Reply:
Was a bad example test, I've just updated to something simpler :)